src/features/domain-settings/Steps/Footers/CompleteStepFooter.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC } from 'react';
+
+import { steps, ConfirmActionType } from '../../DomainSettings.types';
+
+import { FormErrorText } from '../../../ui';
+import { Button, Step } from '@zero-tech/zui/components';
+import { InfoTooltip } from '@zero-tech/zui/components/InfoTooltip';
+import { IconLock1, IconLockUnlocked1 } from '@zero-tech/zui/components/Icons';
+
+import styles from './Footer.module.scss';
+
+interface CompleteStepFooterProps {
+	errorText: string;
+	confirmActionType: ConfirmActionType;
+	onConfirmActionUpdate: (action: ConfirmActionType) => void;
+	onStepUpdate: (step: Step) => void;
+	onLockMetadataStatus: () => void;
+	onClose: () => void;
+}
+
+export const CompleteStepFooter: FC<CompleteStepFooterProps> = ({
+	errorText,
+	confirmActionType,
+	onConfirmActionUpdate,
+	onStepUpdate,
+	onLockMetadataStatus,
+	onClose,
+}) => {
+	const onEdit = () => onStepUpdate(steps[0]);
+
+	return (
```

Comment:
Same story as previous file, could break this JSX down into some subcomponents to make it easier to maintain.

Reply:
Nice, done and will do for the other footer.

Reply:
Nicely done